src/native/http_connection_manager.c

```diff
@@ -51,7 +51,8 @@ static void s_destroy_manager_binding(struct http_connection_manager_binding *bi
         return;
     }
 
-    JNIEnv *env = aws_jni_get_thread_env(binding->jvm);
+    /********** JNI ENV ACQUIRE **********/
+    JNIEnv *env = aws_jni_acquire_thread_env(binding->jvm);
```

Comment:
`s_destroy_manager_binding()` is a helper function, not a callback, we could just pass `env` in as an argument